api/types/events/oneof.go

```diff
@@ -323,6 +331,10 @@ func FromOneOf(in OneOf) (AuditEvent, error) {
 		return e, nil
 	} else if e := in.GetBillingCardDelete(); e != nil {
 		return e, nil
+	} else if e := in.GetLockCreate(); e != nil {
```

Comment:
(pre-existing) It looks like this could benefit from being a type assertion switch (instead of an if/else chain). Out of scope for your PR, though.